### ISSUE TITLE
Use automotive sig coprs for packit tasks

### ISSUE THREAD

**Issue by lsm5** [containers/qm#136]
> > > As a first step, maybe it's worth switching Packit copr tasks to use the automotive-sig coprs in https://github.com/containers/qm/blob/main/.packit.yaml . Currently, it uses `rhcontainerbot/packit-builds` and `rhcontainerbot/qm` coprs.
> > 
> > 
> > @lsm5 are you looking to this task? Should we create an issue in GH?
> 
> @dougsland I'd rather someone else more active on qm take the lead here.

@lsm5 could you please just open an issue against QM with your view and steps which we should approach? I can take the lead to make it happen.

_Originally posted by @dougsland in https://github.com/containers/qm/issues/122#issuecomment-1653756377_

**Comment by lsm5** [comment]
1. Create a couple of coprs in the automotive sig group. One for running builds to test every PR, and another for builds after a PR is committed to main.
2. Update .packit.yaml to use those 2 coprs for `trigger: pull_request` and `trigger: commit` jobs.

**Comment by lsm5** [comment]
@dougsland does that help ^ ?

**Comment by dougsland** [comment]
> @dougsland does that help ^ ?

Sure, it's a start. :-)

**Comment by dougsland** [comment]
@pypingou could you please help us with the automotive sig ?

**Comment by dougsland** [comment]
@Yarboa could you please review this one as you are leading the packit effort now ?

**Comment by Yarboa** [comment]
Sure, let me check

**Comment by lsm5** [comment]
> 1. Create a couple of coprs in the automotive sig group. One for running builds to test every PR, and another for builds after a PR is committed to main.
> 2. Update .packit.yaml to use those 2 coprs for `trigger: pull_request` and `trigger: commit` jobs.

Update: it's probably better to `create` just 1 copr in the automotive sig to host packages built after commit to main branch. For `trigger: pull_request` tasks, you can just use the ephemeral copr that packit creates.

**Comment by Yarboa** [comment]
@dougsland 
looking here https://copr.fedorainfracloud.org/groups/g/centos-automotive-sig/coprs/
I do not see qm copr, 
Once you create qm project it is easy to change owner and project in packit.yaml

**Comment by dougsland** [comment]
@sandrobonazzola do you have the permission to create QM under centos-automotive-sig/coprs?

**Comment by sandrobonazzola** [comment]
Yes, i have permission. Do you need a new repo or do you think `qm` can fit into any of the existing ones?

**Comment by dougsland** [comment]
> Yes, i have permission. Do you need a new repo or do you think `qm` can fit into any of the existing ones?

I would go with a new one but I am okay with existing one. What's your suggestion?

**Comment by dougsland** [comment]
> So, are you going to use bluechi-snapshot or do you want me to create qm-snapshot for you?

Could you please create qm-snapshot? I believe for now, access will be required for me (devel), yariv (qe) and you as centos maint.

**Comment by sandrobonazzola** [comment]
COPR repository created, please request permission for it at https://copr.fedorainfracloud.org/coprs/g/centos-automotive-sig/qm-snapshot/permissions/

**Comment by dougsland** [comment]
> COPR repository created, please request permission for it at https://copr.fedorainfracloud.org/coprs/g/centos-automotive-sig/qm-snapshot/permissions/

requested, thanks a lot @sandrobonazzola ! Kudos! 

@Yarboa PTAL

**Comment by lsm5** [comment]
@sandrobonazzola @Yarboa make sure that `github.com/containers/qm` is added to `Packit allowed forge projects` in the new copr's settings. That should be done before the packit.yaml update PR is merged.

**Comment by lsm5** [comment]
Great, I'll remove the qm package from podman-next copr. Thanks!

**Comment by Yarboa** [comment]
@lsm5 before doing that
Please take a look, maybe you can track the reason why the requests are pending too long?
https://github.com/containers/qm/runs/47343065060

**Comment by Yarboa** [comment]
Done
